### DISCUSSION TITLE
How to automerge an existing PR instead of updating it?

### DISCUSSION THREAD

**Discussion by AndreyNautilus** [renovatebot/renovate#20951] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest (via renovatebot/github-action@v36)

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

When renovate runs and a Pull Request for a group already exists and is ready to be automerged, merge that PR and create a new one, instead of updating the existing PR.

I have a bespoke lock file which I update via 'regex manager'. The lockfile contains more than a hundred of package references, and these packages are developed inside our company, so new versions are published quite frequently (every hour or even more often). The build of the project (Pull Request verification) takes about 1,5 hours, in other words, when a PR is verified there is almost always a new update available, which breaks "automerge" feature.
Is there a way to configure renovate in such a way that instead of updating an existing PR, renovate merges the existing PR first and creates a new PR with new updates?

### What you tried so far.

I tried to use schedules. Since I'm running renovate self-hosted, I configured the cron trigger to be every 2 hours (project build take approximately 1,5 hours, so there is no point in running renovate more often). I grouped all dependencies in my bespoke lock file in one group and update it every run of renovate (other dependencies are scheduled to be updated once a day).
That still doesn't help: the PR is updated and not automerged in almost all cases.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment]
use platform automerge feature of GitHub.

renovate will always try to update the PR.
